frontend/www/js/omegaup/components/arena/ContestListv2.vue

```diff
@@ -345,6 +356,7 @@ export default class ArenaContestList extends Vue {
   @Prop() sortOrder!: ContestOrder;
   @Prop() filterBySignedUp!: boolean;
   @Prop() filterByRecommended!: boolean;
+  @Prop() filterByAll!: boolean;
```

Comment:
The other error I found in the checks is because you are not setting a default value for this property. There are two way to fix it. 
- The first one is adding the default value here:
```suggestion
  @Prop({ default: true }) filterByAll!: boolean;
```
- Or in `contest_listv2.ts` file, you can add:
```js
  let filterByAll: boolean = true;
```
in the line 32. And then you need to add it to the properties object:
```js

        props: {
          contests: this.contests,
          query: this.query,
          tab,
          page,
          sortOrder,
          filterBySignedUp,
          filterByRecommended,
+          filterByAll,
        },
```